src/ir_Kelvinator.h

```diff
@@ -103,6 +103,17 @@ const uint8_t kKelvinatorMinTemp = 16;   // 16C
 const uint8_t kKelvinatorMaxTemp = 30;   // 30C
 const uint8_t kKelvinatorAutoTemp = 25;  // 25C
 
+const uint8_t kKelvinatorSwingLastPos    = 0b0000;  // 0
+const uint8_t kKelvinatorSwingAuto       = 0b0001;  // 1
+const uint8_t kKelvinatorSwingUp         = 0b0010;  // 2
+const uint8_t kKelvinatorSwingMiddleUp   = 0b0011;  // 3
+const uint8_t kKelvinatorSwingMiddle     = 0b0100;  // 4
+const uint8_t kKelvinatorSwingMiddleDown = 0b0101;  // 5
+const uint8_t kKelvinatorSwingDown       = 0b0110;  // 6
+const uint8_t kKelvinatorSwingDownAuto   = 0b0111;  // 7
+const uint8_t kKelvinatorSwingMiddleAuto = 0b1001;  // 9
+const uint8_t kKelvinatorSwingUpAuto     = 0b1011;  // 11
```

Comment:
Please use something like:
https://github.com/crankyoldgit/IRremoteESP8266/blob/9228143207ab7e2968a42d9a92e246237af7bb83/src/ir_Tcl.h#L111-L117
i.e. Not "Up" or "Down", but High/Low etc.
Also, Change "Swing" to "SwingV" in case we need to add a SwingH positions later.